docs/data/frontier-squid.md

```diff
@@ -80,7 +80,7 @@ To configure the Frontier Squid service itself:
 1.  Follow the
     [Configuration section of the upstream Frontier Squid documentation](https://twiki.cern.ch/twiki/bin/view/Frontier/InstallSquid#Configuration).
 2.  Enable, start, and test the service (as described below).
-3.  Enable WLCG monitoring as described in the
+3.  If your site is part of the WLCG, enable WLCG monitoring as described in the
     [upstream documentation on enabling monitoring](https://twiki.cern.ch/twiki/bin/view/Frontier/InstallSquid#Enabling_monitoring)
```

Comment:
Please separate WLCG monitoring and OIM registration into separate bullets. The way it's worded now sounds like you should only register in OIM if you're a WLCG site, which is not the case!

Reply:
ping